src/System.Collections.Specialized/src/System/Collections/Specialized/ListDictionary.cs

```diff
@@ -14,10 +14,10 @@ namespace System.Collections.Specialized
     [Serializable]
     public class ListDictionary : IDictionary
     {
-        private DictionaryNode _head;
-        private int _version;
-        private int _count;
-        private readonly IComparer _comparer;
+        private DictionaryNode head; // Do not rename (binary serialization)
```

Comment:
I wonder if `// Do not rename or change type (binary serialization)` might be better, given your note that CookieCollection changed to List<>.?